### PR TITLE
Allow for configuring API URLs with .env

### LINKED ISSUE #4: Add .env option for custom piston instance

### PR DESCRIPTION
**Pull request by zunda-arrow**
Adds the title. Does work to make that nice.

closes #4

### CODE REVIEW

File: bot/config.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+
+@dataclasses.dataclass
+class Config:
+    TOKEN: str
+    PISTON_URL: str
+    GODBOLT_URL: str
+
+
+env_vars: typing.Any = dotenv.dotenv_values()
```
Comment (circuitsacul) [inline bot/config.py]: whoah how did I not know about this

also can we use `dict[str, str]` and just cast the result if needed?

File: bot/config.py [hunk 2]
```diff
@@ -0,0 +1,15 @@
+import dataclasses
+import typing
```
Comment (circuitsacul) [inline bot/config.py]: For consistency across the project, use `import typing as t`

**Review by circuitsacul — CHANGES_REQUESTED** [review]

File: bot/config.py (hunk 2 above)
Comment (zunda-arrow) [reply bot/config.py]: Done

File: bot/config.py (hunk 1 above)
Comment (zunda-arrow) [reply bot/config.py]: Done

**Review by circuitsacul — APPROVED** [review]

**Pull request merged**